lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -301,21 +302,19 @@ class ImageElements extends Gatherer {
       return map;
     }, /** @type {Object<string, LH.Artifacts.NetworkRequest>} */ ({}));
 
-    const expression = `(function() {
-      ${pageFunctions.getElementsInDocumentString}; // define function on page
-      ${pageFunctions.getBoundingClientRectString};
-      ${pageFunctions.getNodeDetailsString};
-      ${getClientRect.toString()};
-      ${getPosition.toString()};
-      ${getHTMLImages.toString()};
-      ${getCSSImages.toString()};
-      ${collectImageElementInfo.toString()};
-
-      return collectImageElementInfo();
-    })()`;
-
-    /** @type {Array<LH.Artifacts.ImageElement>} */
-    const elements = await driver.evaluateAsync(expression);
+    const elements = await driver.evaluate(collectImageElementInfo, {
+      args: [],
+      deps: [
+        pageFunctions.getElementsInDocumentString,
+        pageFunctions.getBoundingClientRectString,
+        pageFunctions.getNodeDetailsString,
+        getClientRect,
+        getPosition,
+        getHTMLImages,
+        getCSSImages,
+        collectImageElementInfo,
```

Comment:
```suggestion
        getCSSImages,
```
Extra dep not needed

Reply:
👀